CHANGELOG.md

```diff
@@ -1,4 +1,35 @@
-This is the C SDK for EdgeX device services.
+<a name="EdgeX Device SDK C (found in device-sdk-c) Changelog"></a>
+## EdgeX Device SDK C
+[Github repository](https://github.com/edgexfoundry/device-sdk-c)
+
+## [v3.1.0] Napa - 2023-11-15 (Only compatible with the 3.x releases)
+
+### ✨  Features
+
+- Add command line/environment flag for commonConfig ([#486](https://github.com/edgexfoundry/device-sdk-c/issues/486)) ([753c0e6…](https://github.com/edgexfoundry/device-sdk-c/commit/753c0e66273d645c82de476ca0a438ee98891f55))
+- Load common configuration from config provider ([#484](https://github.com/edgexfoundry/device-sdk-c/issues/484)) ([4c52de3…](https://github.com/edgexfoundry/device-sdk-c/commit/4c52de37db7d3bd0277fba2739b75d0a1bd475b3))
+```text
+
+BREAKING CHANGE: load common configuration from config provider ([#484](https://github.com/edgexfoundry/device-sdk-c/issues/484))
```

Comment:
This shouldn't be a breaking change. One could still have all the configuration in the local private config.

Reply:
the default behavior is not load the all configuration from local.
it's the problem we didn't accomplish this feature C v3.0
I am removing the breaking change anyway.